### ISSUE TITLE
Authorization header

### ISSUE THREAD

**Issue by trev** [Mermade/widdershins#76]
WDYT in having something like:

openapi3.js around line 136

```javascript
    if ('bearerAuth' in data.api.components.securitySchemes) {
        var authorization = {};
        authorization.name = 'Authorization';
        authorization.type = 'string';
        authorization.in = 'header';
        authorization.exampleValues = {};
        authorization.exampleValues.json = "'Bearer {access_token}'";
        authorization.exampleValues.object = ['Bearer {access_token}'];
        data.allHeaders.push(authorization);
    }
```

That way the code examples would be more complete since the generated header would include the required authorization header.

e.g

```shell
curl -X POST https://x.com/activate \
  -H 'Accept: application/json' \
  -H 'Authorization: Bearer {access_token}'
```

instead of

```shell
curl -X POST https://x.com/activate \
  -H 'Accept: application/json'
```

**Comment by MikeRalphson** [comment]
I like it! Do you want to work up a PR based on the code above? I think you'd need to detect the `type: http` and `scheme: bearer` of the securityScheme referenced by the operation or the default [Security Requirement Object](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.1.md#securityRequirementObject).

**Comment by trev** [comment]
Sure, I'll give it a dig early in the new year. Been a while since I've JS'ed :)

**Comment by alexanderkjeldaas** [comment]
This looks promising! Any progress?

**Comment by trev** [comment]
Sorry @alexanderkjeldaas - I'm not likely to get to this for a while as I have other commitments at the moment.

**Comment by MikeRalphson** [comment]
Thanks for the update @trev - I should be able to pick this up.

**Comment by MikeRalphson** [comment]
@trev @alexanderkjeldaas @luigi-riefolo I've added code to generate an `Authorization` header where the securityScheme is either `http` with the `bearer` scheme, or `auth2` or `openIdConnect`.

If you can test from the GitHub repository, that would be great, otherwise these changes will be published as part of Widdershins v3.1.0

**Comment by luigi-riefolo** [comment]
will do

**Comment by MikeRalphson** [comment]
Included in v3.1.0 - please chase this issue with any feedback/problems.
